### PR TITLE
feat: initial laravel 9 compatibility

### PR DESCRIPTION
**Pull request by divine**
Todo:

- [x] PHP-CS-Fixer PHP v8
- [ ] PHPStan?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
-        "illuminate/support": "^8.0",
-        "illuminate/container": "^8.0",
-        "illuminate/database": "^8.0",
-        "illuminate/events": "^8.0",
-        "mongodb/mongodb": "^1.6"
+        "illuminate/support": "9.x-dev",
+        "illuminate/container": "9.x-dev",
+        "illuminate/database": "9.x-dev",
+        "illuminate/events": "9.x-dev",
+        "mongodb/mongodb": "^1.11"
```
Comment (Wernke96) [inline composer.json:26]: Can you take the dev tag for this since laravel 9 is out?

**Review by Wernke96 — CHANGES_REQUESTED** [review]
Sorry about my pull request early I didn't see your message.

File: composer.json (hunk 1 above)
Comment (divine) [reply composer.json:26]: Of course but we won't be able to merge it as we don't have enough permissions to do so.

Thanks!

**Review by Smolevich — APPROVED** [review]

**Review by Smolevich — APPROVED** [review]

**Review by Giacomo92 — APPROVED** [review]

**Comment by divine** [comment]
@jenssegers can you reset the required status check?

Thanks!

**Review by necessarylion — APPROVED** [review]

**Comment by saeed-darestany** [comment]
@divine  🤔

**Review by Wernke96 — APPROVED** [review]

**Comment by saeed-darestany** [comment]
@jenssegers 
:walking:

**Comment by jenssegers** [comment]
@divine Should be ok now!

**Review by jenssegers — APPROVED** [review]

**Pull request merged**